app.py

```diff
@@ -392,10 +430,24 @@ def add_new_alert(ref_id: str, camera_id: str):
     result = db.alerts.insert_one(alert)
     return result
 
+@app.route('/alerts/<ref_id>/<email>', methods=['GET'])
+def alert_email(ref_id: str, email: str):
+    a = db.alerts.find_one({"ref_id": ref_id})
+    print("RESULTS: "+str(a))
+    c = get_camera_by_id(a['camera_id'])
+    p = db.people.find({"ref_id": ref_id})
+    db.unauthorized
+    for i in p:
+        print("Image:" +str(i))
+        # get_person_by_id({"ref_id": ObjectId(ref_id)})
+    send_alert_email(a, p.img, c, email)
+    return True
```

Comment:
`send_alert_email()` returns boolean, so you should set a variable like `result = send_alert_email(a, p.img, c, email)`, so if result == true, return response with status code 200 so the frontend knows the email was sent and if result == false send code 500 or something so the frontend knows it failed